### PR TITLE
added converter for additional instructions

### PR DESCRIPTION
**Pull request by phillip-whitaker-hmcts**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/EUI-5015

### Change description ###

Added additional instructions to CYA page 

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/hearings/converters/additional-instructions.converter.ts [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import {Observable} from 'rxjs';
+import {map} from 'rxjs/operators';
+import {State} from '../store';
+import {AnswerConverter} from './answer.converter';
+
+export class AdditionalInstructionsConverter implements AnswerConverter {
```
Comment (rayliangatos) [inline src/hearings/converters/additional-instructions.converter.ts:6]: Could we rename the file to additional-instructions.answer.converter and this to AdditionalInstructionsAnswerConverter? This is to consistent with other converters.

Comment (phillip-whitaker-hmcts) [reply src/hearings/converters/additional-instructions.converter.ts:6]: Sure, that wasn't the naming convention when I did the work

**Review by rayliangatos — APPROVED** [review]
Fab, approved

File: src/hearings/converters/additional-instructions.converter.spec.ts [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import {cold} from 'jasmine-marbles';
+import {of} from 'rxjs';
+import {initialState} from '../hearing.test.data';
+import {State} from '../store';
+import {AdditionalInstructionsAnswerConverter} from './additional-instructions.answer.converter';
+
+describe('AdditionalInstructionsAnswerConverter', () => {
```
Comment (rayliangatos) [inline src/hearings/converters/additional-instructions.converter.spec.ts:7]: probably this file needs to rename to additional-instructions.answer.converter.spec.ts?

File: src/hearings/hearing.test.data.ts [hunk 3]
```diff
@@ -1123,7 +1123,7 @@ export const initialState = {
             'immigrationDetentionCentre',
             'inCameraCourt'
           ],
-          listingComments: null,
+          listingComments: 'blah blah blah',
```
Comment (kuda1992) [inline src/hearings/hearing.test.data.ts:1126]: 😊😊😊😊 I love the creativity 😂😂

**Review by kuda1992 — APPROVED** [review]
Looks good to me

**Pull request merged**